scripts/set-up-environment.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+VERSION="14.3"
```

Comment:
```suggestion
XCODE_VERSION="14.3"
```

to improve clarity.

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/188/commits/38e2abf558598652ce15ab27ffdd5a565fea4cfd